metaflow/__init__.py

```diff
@@ -219,43 +219,43 @@ def _can_handle_orig_module():
 # Now override everything other than modules
 __version_addl__ = None
 try:
-    import metaflow_custom.toplevel.toplevel as extension_module
+    import metaflow_extensions.toplevel.toplevel as extension_module
 except ImportError as e:
     ver = sys.version_info[0] * 10 + sys.version_info[1]
     if ver >= 36:
         # e.name is set to the name of the package that fails to load
         # so don't error ONLY IF the error is importing this module (but do
         # error if there is a transitive import error)
         if not (isinstance(e, ModuleNotFoundError) and \
-                e.name in ['metaflow_custom', 'metaflow_custom.toplevel',
-                           'metaflow_custom.toplevel.toplevel']):
+                e.name in ['metaflow_extensions', 'metaflow_extensions.toplevel',
+                           'metaflow_extensions.toplevel.toplevel']):
             print(
-                "Cannot load metaflow_custom top-level configuration -- "
-                "if you want to ignore, uninstall metaflow_custom package")
+                "Cannot load metaflow_extensions top-level configuration -- "
+                "if you want to ignore, uninstall metaflow_extensions package")
             raise
 else:
     # We load into globals whatever we have in extension_module
     # We specifically exclude any modules that may be included (like sys, os, etc)
-    # *except* for ones that are part of metaflow_custom (basically providing
+    # *except* for ones that are part of metaflow_extensions (basically providing
     # an aliasing mechanism)
     lazy_load_custom_modules = {}
     addl_modules = extension_module.__dict__.get('__mf_promote_submodules__')
     if addl_modules:
-        # We make an alias for these modules which the metaflow_custom author
+        # We make an alias for these modules which the metaflow_extensions author
         # wants to expose but that may not be loaded yet
         lazy_load_custom_modules = {
-            'metaflow.%s' % k: 'metaflow_custom.%s' % k for k in addl_modules}
+            'metaflow.%s' % k: 'metaflow_extensions.%s' % k for k in addl_modules}
     for n, o in extension_module.__dict__.items():
         if not n.startswith('__') and not isinstance(o, types.ModuleType):
             globals()[n] = o
         elif isinstance(o, types.ModuleType) and o.__package__ and \
-                o.__package__.startswith('metaflow_custom'):
+                o.__package__.startswith('metaflow_extensions'):
             lazy_load_custom_modules['metaflow.%s' % n] = o
     if lazy_load_custom_modules:
         # Prepend to make sure custom package overrides things
         sys.meta_path = [_LazyLoader(lazy_load_custom_modules)] + sys.meta_path
 
-    __version_addl__ = getattr(extension_module, '__mf_customization__', '<unk>')
+    __version_addl__ = getattr(extension_module, '__mf_extensions__', '<unk>')
```

Comment:
@romain-intel This change may require a corresponding change in the nflx-internal metaflow-extensions.

Reply:
done